/**:
  ros__parameters:
    max_range: 30.0
    downsample_num: 1081
    distance_threshold: 0.033

    model_name: LidarGCN #LidarGCN or LidarGAT LidarGCNLstm LidarGATLstm
    model_path: .pth
    input_dim: 2
    hidden_dim: 128
    output_dim: 2
    pool_method: mean # mean or max

